e2e/tests/apiproduct-crud.spec.ts

```
@@ -415,8 +415,13 @@ spec:
     await expect(createButton).toBeEnabled({ timeout: 10000 });
     await createButton.click();
 
-    // ResourceYAMLEditor navigates to the k8s details page after creation (not our custom page).
-    // Use a full page.goto() so the console properly loads our custom list route.
+    // Wait for ResourceYAMLEditor to finish creating and navigate to the details page
+    await expect(page).toHaveURL(
```

ESLint

[error] Insert `··`

(prettier/prettier)